servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpExecutionStrategies.java

```diff
@@ -51,20 +51,33 @@ public static HttpExecutionStrategy defaultStrategy() {
      * paths. When merged with another execution strategy the result is always this strategy.
      *
      * @return {@link HttpExecutionStrategy} that disables all request-response path offloads.
-     * @see #anyStrategy()
+     * @see #offloadNone()
+     * @deprecated Replaced with more descriptive {@link #offloadNever()}.
      */
-    public static HttpExecutionStrategy noOffloadsStrategy() {
+    @Deprecated
+    public static HttpExecutionStrategy noOffloadStrategy() {
+        return OFFLOAD_NEVER_STRATEGY;
+    }
+
+    /**
+     * A special {@link HttpExecutionStrategy} that disables all offloads on the request-response and transport event
+     * paths. When merged with another execution strategy the result is always this strategy.
```

Comment:
Consider clarifying that it should not be used for `requireOffloads()`. Add the same clarification for `defaultStrategy()` and point to the appropriate alternatives.